src/i18n/fr_FR/buildings.ts

```diff
@@ -64,6 +64,20 @@ export default {
                 coins: 25,
                 duration: '5 jours',
             },
+            {
+                caption: 'Grande caserne de pompiers',
+                credits: 1_000_000,
+                coins: 50,
+                duration: '7 jours',
+                maxExtensionsFunction: (
+                    buildingsByType: Record<number, Building[]>
+                ): number => Math.floor((buildingsByType[0]?.length ?? 0) / 10),
```

Comment:
In the case of the fire department, police and ambulance extensions, the small stations also count in the count. you have now only added the normal stations to the count here.

Reply:
ok thanks I'll send the corrections